### PR TITLE
fail2ban 0.10.5

### PR DESCRIPTION
**Pull request by sayomaki**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by alebcay** [comment]
```
==> brew test fail2ban --verbose
==> FAILED
Testing fail2ban
/usr/bin/sandbox-exec -f /private/tmp/homebrew20200111-66351-1ncqroa.sb ruby -W0 -I $LOAD_PATH -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/fail2ban.rb --verbose
==> /usr/local/Cellar/fail2ban/0.10.5/bin/fail2ban-client --test
2020-01-11 17:28:08,788 fail2ban                [66365]: ERROR   Failed during configuration: Bad value substitution:
	section: [sshd]
	option : backend
	key    : sshd_backend
	rawval : 

2020-01-11 17:28:08,789 fail2ban                [66365]: ERROR   ERROR: test configuration failed
Error: fail2ban: failed
Failed executing:
/usr/local/Homebrew/Library/Homebrew/formula.rb:1932:in `block in system'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1869:in `open'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1869:in `system'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/fail2ban.rb:129:in `block in <class:Fail2ban>'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1745:in `block (3 levels) in run_test'
/usr/local/Homebrew/Library/Homebrew/utils.rb:478:in `with_env'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1744:in `block (2 levels) in run_test'
/usr/local/Homebrew/Library/Homebrew/formula.rb:865:in `with_logging'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1743:in `block in run_test'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1982:in `block in mktemp'
/usr/local/Homebrew/Library/Homebrew/mktemp.rb:57:in `block in run'
/usr/local/Homebrew/Library/Homebrew/mktemp.rb:57:in `chdir'
/usr/local/Homebrew/Library/Homebrew/mktemp.rb:57:in `run'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1981:in `mktemp'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1737:in `run_test'
/usr/local/Homebrew/Library/Homebrew/test.rb:40:in `block in <main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:93:in `block in timeout'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:33:in `block in catch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:33:in `catch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:33:in `catch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/timeout.rb:108:in `timeout'
/usr/local/Homebrew/Library/Homebrew/test.rb:39:in `<main>'
```

**Comment by willi123yao** [comment]
Might be related to setup configuration as mentioned in [here](https://github.com/fail2ban/fail2ban/issues/2049#issuecomment-384345748), will be replacing that line of configuration during installation.

**Comment by fxcoudert** [comment]
Can we build this with `depends_on "python@3.8"`?

**Comment by willi123yao** [comment]
Lets try that, should be working if we change the references to python 3.8.

File: Formula/fail2ban.rb [hunk 1]
```diff
@@ -11,11 +11,12 @@ class Fail2ban < Formula
   end
 
   depends_on "help2man" => :build
+  depends_on "python@3.8"
```
Comment (fxcoudert) [inline Formula/fail2ban.rb]: This will complain: alphabetical order within category, with all build deps first, then runtime

Comment (willi123yao) [reply Formula/fail2ban.rb]: Yes, I have realised that as well. Apparently I did a mistake by putting `python@3.8` as a build dependency only

**Pull request closed without merging**